src/Hacks/autodefuse.cpp

```diff
@@ -54,10 +54,10 @@ void AutoDefuse::CreateMove(CUserCmd *cmd)
 	else
 	if (Settings::AutoDefuse::enabled)
 	{	
-		if (localplayer->HasDefuser() && bombTimer > 5.5f)
+		if (localplayer->HasDefuser() && bombTimer > 5.039f)
```

Comment:
what are these magic numbers

Reply:
It gives 0.000 when you defuse. (Only tested on 50 ping 128 tick)